concepticondata/conceptlists/Chen-2012-888.tsv

```diff
@@ -686,7 +686,7 @@ Chen-2012-888-684	684	剁	chop into pieces	794	2149	CHOP (INTO PIECES)		1296	动
 Chen-2012-888-685	685	翻	turn over (on bed)	795	3537	TURN OVER		1310	动词
 Chen-2012-888-686	686	发烧	have a fever	795	2713	HAVE A FEVER		1305	动词
 Chen-2012-888-687	687	纺	spin (into yarn)	795	121	SPIN		1313	动词
-Chen-2012-888-688	688	放（牛）	put out to pasture	796				1316	动词
+Chen-2012-888-688	688	放（牛）	put out to pasture	796	2031	PUT OUT TO PASTURE		1316	动词
```

Comment:
In [Castro-2015-608-383](https://concepticon.clld.org/values/Castro-2015-608-383), the same gloss is mapped and translated as GRAZE. @LinguList is it possible to disambiguate the two?

Reply:
In this case, it is like "put (a cow)", so it means probably something like "put out to pasture". GRAZE is not causative in our definitoin, but this "put to pasture" is like "grasen lassen", so I'd make the disambiguation along this line. This is definitely causative here.

Reply:
So, @LinguList, should I leave it as 2031 PUT OUT TO PASTURE? Also, if the same gloss is mapped to GRAZE in other cases, should I open a new issue to discuss this?

Reply:
Leave it for now, I think that it is close enough.